components/Header/StyledHeader.styled.js

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from "styled-components";
```

Comment:
do you need to import `{css}` here?

```suggestion
import styled from "styled-components";
```